lightly/api/api_workflow_upload_dataset.py

```diff
@@ -39,7 +39,7 @@
     from lightly.data import LightlyDataset
     _lightly_dataset_available = True
 except (
-    OSError,      # Different CUDA versions for torch and torchvision
+    RuntimeError,         # Different CUDA versions for torch and torchvision
```

Comment:
`torchvision` raises a `RuntimeError` not a `OSError`.

Reply:
```
OSError: libcudnn.so.8: cannot open shared object file: No such file or directory
Traceback (most recent call last):
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/pip_deps_sematic/site-packages/sematic/resolvers/worker.py", line 159, in main
    func: Calculator = run.get_func()  # type: ignore
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/pip_deps_sematic/site-packages/sematic/db/models/run.py", line 231, in get_func
    func = getattr(importlib.import_module(import_path), func_name)
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/python_x86_64-unknown-linux-gnu/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/voxel/core/ml/data/curation/lib/lightly_worker.py", line 31, in <module>
    from lightly.api import ApiWorkflowClient
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/pip_deps_lightly/site-packages/lightly/__init__.py", line 110, in <module>
    import torchvision.models.vision_transformer
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/pip_deps_torchvision/site-packages/torchvision/__init__.py", line 4, in <module>
    import torch
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/pip_deps_torch/site-packages/torch/__init__.py", line 217, in <module>
    _load_global_deps()
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/pip_deps_torch/site-packages/torch/__init__.py", line 177, in _load_global_deps
    raise err
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/pip_deps_torch/site-packages/torch/__init__.py", line 172, in _load_global_deps
    ctypes.CDLL(lib_path, mode=ctypes.RTLD_GLOBAL)
  File "/app/core/infra/sematic/perception/data_ingestion/door_data/main_lightly-image_image.binary.runfiles/python_x86_64-unknown-linux-gnu/lib/python3.9/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libcudnn.so.8: cannot open shared object file: No such file or directory
```
This is stacktrace of what we are seeing. Failing even with this

Reply:
Thank you, @diksha. I'll include the `OSError` again.